velox/common/memory/HashStringAllocator.cpp

```diff
@@ -616,4 +616,20 @@ void HashStringAllocator::checkConsistency() const {
   VELOX_CHECK_EQ(bytesInFreeList, freeBytes_);
 }
 
+void HashStringAllocator::checkEmpty() const {
+  for (auto i = 0; i < kNumFreeLists - 1; ++i) {
+    VELOX_CHECK(free_[i].empty());
```

Comment:
Perhaps, add an error message to help debugging. For example, it might be helpful to include 'i' in the error message and perhaps the size of the free list.